tests/api/test_bulk_delete.py

```diff
@@ -286,6 +286,46 @@ def test_delete_draft_orders(
         id__in=[order.id for order in orders]).count() == len(orders)
 
 
+def test_delete_draft_orders_lines(
+        staff_api_client, order_with_lines, permission_manage_orders):
+    order = order_with_lines
+    order_lines = [line for line in order]
+    order.status = OrderStatus.CANCELED
```

Comment:
Do you need this field?
I don't see a place where you are saving this change to DB